### PR TITLE
Set up ISV profile pages

### PR DESCRIPTION
**Pull request by anthonydillon**
The initial routing and skeleton for the ISV profile component.

### CODE REVIEW

**Comment by anthonydillon** [comment]
## QA
- Go to http://0.0.0.0:6543/isv/:flags:/blues/
- Check you see a panel with the title "ISV Profile"
- Go to http://0.0.0.0:6543/isv/
- Not check just the GUI renders

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5623/
Test PASSed.

**Comment by hatched** [comment]
Thanks @anthonydillon looks good

**Comment by huwshimi** [comment]
:+1: here too. No QA.

File: jujugui/static/gui/src/app/components/isv-profile/isv-profile.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+/*
+This file is part of the Juju GUI, which lets users view and manage Juju
+environments within a graphical interface (https://launchpad.net/juju-gui).
+Copyright (C) 2015 Canonical Ltd.
```
Comment (frankban) [inline jujugui/static/gui/src/app/components/isv-profile/isv-profile.js]: 2016

File: jujugui/static/gui/src/app/components/isv-profile/isv-profile.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+This program is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
+General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License along
+with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
```
Comment (frankban) [inline jujugui/static/gui/src/app/components/isv-profile/isv-profile.js]: Please add a comment explaining the usage and goals of this component.

File: jujugui/static/gui/src/app/components/isv-profile/isv-profile.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+'use strict';
+
+YUI.add('isv-profile', function() {
+
+  juju.components.ISVProfile = React.createClass({
```
Comment (frankban) [inline jujugui/static/gui/src/app/components/isv-profile/isv-profile.js]: Please add a docstring for this component as well.

File: jujugui/static/gui/src/app/components/isv-profile/test-isv-profile.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
+/*
+This file is part of the Juju GUI, which lets users view and manage Juju
+environments within a graphical interface (https://launchpad.net/juju-gui).
+Copyright (C) 2015 Canonical Ltd.
```
Comment (frankban) [inline jujugui/static/gui/src/app/components/isv-profile/test-isv-profile.js]: 2016

**Comment by frankban** [comment]
:+1: with some minor comments.
I had some problems QAing this: `http://0.0.0.0:6543/isv/` renders the GUI as usual, without any ISV related stuff. I believe that's what you wanted to be QAed there. However `http://0.0.0.0:6543/isv/:flags:/blues/` instead renders the following:
screenshot from 2016-08-18 13-13-52

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5630/
Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5631/
Test PASSed.

**Comment by hatched** [comment]
Thanks @anthonydillon 

:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**